app/helpers/enterprises_helper.rb

```diff
@@ -88,4 +88,11 @@ def shop_trial_expired?(enterprise)
   def remaining_trial_days(enterprise)
     distance_of_time_in_words(Time.zone.now, enterprise.shop_trial_start_date + Spree::Config[:shop_trial_length_days].days)
   end
+
+  # Enable embeddeding within iframes on external sites.
+  # Call with :after_filter on controller actions.
+  def enable_embedded_shopfront
```

Comment:
We can do iframe testing seperately, but I don't think we can do it directly inside of RSpec.